### PR TITLE
Angular

### PR DESCRIPTION
**Pull request by Kuzless**
Added filter

### CODE REVIEW

File: MyMessenger.Angular/src/app/pages/auth/login/login.component.html
Comment (msyniuk-reenbit) [inline MyMessenger.Angular/src/app/pages/auth/login/login.component.html:1]: Disable the Log In button

File: MyMessenger.Angular/src/app/services/auth.service.ts [hunk 1]
```diff
@@ -1,43 +1,49 @@
...
-    const response = await this.wrapper.postAsync('Auth/', data);
-    if (response && response.accessToken && response.refreshToken) {
-      localStorage.setItem('accessToken', response.accessToken);
-      localStorage.setItem('refreshToken', response.refreshToken);
-      this.loggedIn.next(true);
-      return true;
-    } else {
-      console.error('Invalid login credentials.');
-      return false;
+    try{
```
Comment (msyniuk-reenbit) [inline MyMessenger.Angular/src/app/services/auth.service.ts]: In 99% cases you don't need to use `try catch` syntax in Angular, it has it's separate controls to manage this. 
For example here is your refactored code: 

image

Much more readability

Comment (Kuzless) [reply MyMessenger.Angular/src/app/services/auth.service.ts]: merged branch with this change

File: MyMessenger.Angular/src/app/pages/auth/login/login.component.html
Comment (Kuzless) [reply MyMessenger.Angular/src/app/pages/auth/login/login.component.html:1]: also merged branch with this change

**Pull request merged**